RELEASE_NOTES.md

```diff
@@ -9,6 +9,8 @@
 * Added the ability to automatically copy files into the built images filesystem (see Image Configuration Directory Changes below)
 * Kubernetes manifests are now applied in a systemd service instead of using the `/manifests` directory 
 * Helm chart installation backOffLimit changed from 1000(default) to 20
+* Improved Elemental handling when using SL Micro 6.0
```

Comment:
Can we rephrase this and move it to the API section below? `Elemental configuration now requires a registration code in order to install the necessary RPMs from the official sources` or something similar.